packages/gatsby/src/utils/page-ssr-module/bundle-webpack.ts

```diff
@@ -224,12 +227,14 @@ export async function createPageSSRBundle({
     `utf-8`
   )
 
-  functionCode = functionCode.replace(
-    `%CDN_DATASTORE_PATH%`,
-    shouldBundleDatastore()
-      ? ``
-      : `${state.adapter.config.deployURL ?? ``}/${LmdbOnCdnPath}`
-  )
+  functionCode = functionCode
+    .replace(
+      `%CDN_DATASTORE_PATH%`,
+      shouldBundleDatastore()
+        ? ``
+        : `${state.adapter.config.deployURL ?? ``}/${LmdbOnCdnPath}`
+    )
+    .replace(`%PATH_PREFIX`, pathPrefix)
```

Comment:
Should `%PATH_PREFIX` have a trailing `%` for consistency with the placeholder above?

Reply:
done in https://github.com/gatsbyjs/gatsby/pull/38666/commits/63b5cce83b7caf40df41bdebf8efc5fb3221dd65